tests/utils/test_cache_utils.py

```diff
@@ -479,44 +485,7 @@ def test_static_cache_greedy_decoding_pad_left(self, attn_implementation, cache_
         with self.subTest(f"{attn_implementation}, static, compiled"):
             self.assertListEqual(decoded, EXPECTED_GENERATION)
 
-    @require_torch_gpu
-    @parameterized.expand(
-        [
-            ("eager", "static"),
-            ("sdpa", "static"),
-        ]
-    )
-    def test_static_cache_greedy_decoding_pad_right(self, attn_implementation, cache_implementation):
```

Comment:
generation is not meant to work well with right-padding, no need to spend resources testing it